org.photoqt.PreviewQt.yml

```diff
@@ -0,0 +1,311 @@
+app-id: org.photoqt.PreviewQt
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: previewqt
+finish-args:
+    - --share=ipc
+    - --socket=wayland
+    - --socket=fallback-x11
+    - --filesystem=host
+    - --device=dri
+    - --socket=pulseaudio
```

Comment:
an image viewer need sound access?

Reply:
Oh, it's not an image viewer. It's a preview application for images, videos, documents, and archives, with more to be added going forward. In particular the video support needs sound access.